bridge/pkg/terra/terra.go/msg/types.go

```diff
@@ -0,0 +1,132 @@
+package msg
+
+import sdk "github.com/cosmos/cosmos-sdk/types"
+
+// Types - wrappers to expose interface
```

Comment:
Is this code copied from cosmos-sdk? Why? Needs proper copyright headers and/or notices to comply with their open source license.